scripts/ci/update-open-pr

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```

Comment:
I personally love to have a `set -x` here. This let's the script fail once a process ends with a non-zero exit code. That way we have no silent failures.